endaq/calc/filters.py

```diff
@@ -4,10 +4,50 @@
 import functools
 
 import pandas as pd
+import numpy as np
 import scipy.signal
 
 from endaq.calc import utils
 
+def _get_filter_params(low_cutoff, high_cutoff):
+    """Get the filter type and cutoff frequency array."""
+    cutoff_freqs: Union[float, Tuple[float, float]]
+    filter_type: str
+
+    if low_cutoff is not None and high_cutoff is not None:
+        cutoff_freqs = (low_cutoff, high_cutoff)
+        filter_type = "bandpass"
+    elif low_cutoff is not None:
+        cutoff_freqs = low_cutoff
+        filter_type = "highpass"
+    elif high_cutoff is not None:
+        cutoff_freqs = high_cutoff
+        filter_type = "lowpass"
+    else:
+        filter_type = ""
```

Comment:
I moved this to above the if/else statements but the tests still fail. I think we still want this to be okay because this allows the user to effectively just apply a Tukey window?